packages/components/src/components/notification/notification.stories.ts

```diff
@@ -227,9 +225,11 @@ export const ToastNotification = {
 
 /**
  * Display a toast notification positioned at the bottom-center of the screen by setting the `toastStack` attribute to `bottom-center`. Click on the **Show code** button to see the JavaScript code responsible for generating the toast notification.
+ *
+ * __Hint:__ Click on one of the buttons below to see the corresponding toast notification.
  */
 export const ToastBottomCenter = {
-  name: 'Toast Notification (Bottom Center)',
+  name: ' ',
```

Comment:
If it's empty I guess we can completely remove the name

Reply:
With this blank space it was possible to remove the title of the section and make it more alike figma (see screenshot). The goal was to be able to have two stories in the same section.

<img width="574" alt="image" src="https://github.com/user-attachments/assets/76920818-8333-46d8-a802-73eed225b386">